onos-legacy/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
Add the `---` YAML start of document

Does this pass `helm lint --strict` ?

Reply:
Yeah I just ran it and it passes.